lib/facter/sensu_version.rb

```diff
@@ -0,0 +1,8 @@
+Facter.add(:sensu_version) do
+  confine :kernel => 'Linux'
```

Comment:
This should not confine for Linux as it should work on the other posix platforms.

Perhaps a conditional such as
```
if Facter.value('kernel') == 'windows'
  <GH will figure out windows code>
else
  <posix code here>
end
```